easybuild/easyconfigs/b/Blitz++/Blitz++-1.0.2-foss-2020a.eb

```
@@ -0,0 +1,31 @@
+easyblock = 'CMakeMake'
+
+name = 'Blitz++'
+version = '1.0.2'
+
+homepage = 'https://github.com/blitzpp/blitz'
+
+description = """
+ Blitz++ is a (LGPLv3+) licensed meta-template library for array manipulation
+ in C++ with a speed comparable to Fortran implementations, while preserving an
+ object-oriented interface
+"""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'pic': True}
+
+source_urls = [('https://github.com/blitzpp/blitz/archive/')]
+sources = ['%(version)s.tar.gz']
+checksums = ['500db9c3b2617e1f03d0e548977aec10d36811ba1c43bb5ef250c0e3853ae1c2']
+
+builddependencies = [('CMake', '3.16.4')]
+
+separate_build_dir = True
+
+sanity_check_paths = {
+    'files': ['lib64/libblitz.a', 'lib64/libblitz.so'],
```

We have a constant that we use instead of .so as it can differ one some platforms;
```suggestion
    'files': ['lib64/libblitz.a', 'lib64/libblitz.%s' % SHLIB_EXT],
```

---

Thanks for the feedback @Micket.  I've changed to use `SHLIB_EXT` as you suggested and also changed the toolchain to `GCCcore` since I don't think `foss` was strictly required.